### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by Choiyounsou** [qalabeabbas49/Data-Analyst-Portfolio#1]
Hello,

I am interested in speaker diarization models. I recently trained conversational data using the svoice model, but its performance was not satisfactory. I noticed that others on the svoice issue board seem to have similar concerns, so I was wondering if you have found a solution to this issue.

Thank you!

**Comment by qalabeabbas49** [comment]
Hi, what kind of issue are you facing?

**Comment by Choiyounsou** [comment]
Hello, I am interested in studying speech data and speech models. I trained a model based on the architecture released by svoice, using conversational data(8k data) that contains both overlapping and non-overlapping segments—specifically, recordings of dialogues between two speakers. However, the performance of the completed model, trained on such data, has been very poor. Even after adding more data, the performance did not improve. 
In other words, the problem is that when decoding conversational data, speaker separation does not work well. My goal is to create a model that separates speakers from everyday conversation audio files, and I wanted to ask if there is any potential solution to this problem. Thank you for your time.

**Comment by Choiyounsou** [comment]
config.yaml The hyperparameters of the file are as follows.

defaults:
  - dset: debug
  - hydra/job_logging: colorlog
  - hydra/hydra_logging: colorlog

# Dataset related
sample_rate: 8000
segment: 12
stride: 1    # in seconds, how much to stride between training examples
pad: true   # if training sample is too short, pad it
cv_maxlen: 8
validfull: 1   # use entire samples at valid

# Logging and printing, and does not impact training
num_prints: 5
device: cuda
num_workers: 5
verbose: 0
show: 0   # just show the model and its size and exit

# Checkpointing, by default automatically load last checkpoint
checkpoint: True
continue_best: false
restart: False # Ignore existing checkpoints
checkpoint_file: checkpoint.th
history_file: history.json
samples_dir: samples

# Other stuff
seed: 2036
dummy:  # use this if you want twice the same exp, with a name

# Evaluation stuff
pesq: false # compute pesq?
eval_every: 10
keep_last: 0

# Optimization related
optim: adam
lr: 5e-4 #5e-4
beta2: 0.999
stft_loss: False
stft_sc_factor: .5
stft_mag_factor: .5
epochs: 50
batch_size: 16
max_norm: 51
# learning rate scheduling
lr_sched: step # can be either step or plateau
step: 
  step_size: 2
  gamma: 0.98
plateau:
  factor: 0.5
  patience: 5

# Models
model: swave # either demucs or dwave
swave:
  N: 128
  L: 8
  H: 128
  R: 6
  C: 2
  input_normalize: False

# Experiment launching, distributed
ddp: True
ddp_backend: nccl
rendezvous_file: ./rendezvous

# Internal config, don't set manually
rank:
world_size:

# Hydra config
hydra:
  run:
    dir: ./speaker_separate_exp/exp_${hydra.job.override_dirname}
  job:
    config:
      # configuration for the ${hydra.job.override_dirname} runtime variable
      override_dirname:
        kv_sep: '='
        item_sep: ','
        # Remove all paths, as the / in them would mess up things
        # Remove params that would not impact the training itself
        # Remove all slurm and submit params.
        # This is ugly I know...
        exclude_keys: [
          'hydra.job_logging.handles.file.filename',
          'dset.train', 'dset.valid', 'dset.test', 'dset.mix_json', 'dset.mix_dir',
          'num_prints', 'continue_from',
          'device', 'num_workers', 'print_freq', 'restart', 'verbose',
          'log', 'ddp', 'ddp_backend', 'rendezvous_file', 'rank', 'world_size']
  job_logging:
    handlers:
      file:
        class: logging.FileHandler
        mode: w
        formatter: colorlog
        filename: trainer.log
      console:
        class: logging.StreamHandler
        formatter: colorlog
        stream: ext://sys.stderr

  hydra_logging:
    handlers:
      console:
        class: logging.StreamHandler
        formatter: colorlog
        stream: ext://sys.stderr

**Comment by qalabeabbas49** [comment]
Can you share sample of your dataset and how you prepared it? 

you can contact me directly qalabeabbas49@gmail.com
